### PR TITLE
Updates to Kubernetes example

### PR DESCRIPTION
**Pull request by font**
- Add label and namespace to ceph tests
- Cleanup storage node-type label from all nodes
- Script to just create kube ceph secrets
- Default to use IPv4 even if IPv6 address exists

### CODE REVIEW

File: examples/kubernetes/create_secrets.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```
Comment (leseb) [inline examples/kubernetes/create_secrets.sh]: Don't we need to include this script execution into `create_ceph_cluster.sh`?

File: examples/kubernetes/ceph-mon-v1-ds.yaml [hunk 2]
```diff
@@ -50,7 +50,7 @@ spec:
             - name: KV_TYPE
               value: k8s
             - name: NETWORK_AUTO_DETECT
-              value: "1"
+              value: "4"
```
Comment (leseb) [inline examples/kubernetes/ceph-mon-v1-ds.yaml]: Just for my info, what does `value` determine?

File: examples/kubernetes/delete_ceph_cluster.sh [hunk 3]
```diff
@@ -1,3 +1,4 @@
 #!/bin/bash
 
 kubectl delete namespace ceph
+kubectl label nodes --all node-type-
```
Comment (leseb) [inline examples/kubernetes/delete_ceph_cluster.sh]: anything wrong with that command or is it just me? :)

File: examples/kubernetes/create_secrets.sh (hunk 1 above)
Comment (font) [inline examples/kubernetes/create_secrets.sh]: Sure, we could do that. The intent was just to have a way of creating secrets without creating everything else along with it.

File: examples/kubernetes/ceph-mon-v1-ds.yaml (hunk 2 above)
Comment (font) [inline examples/kubernetes/ceph-mon-v1-ds.yaml]: This determines what address we use for the monitor and for the ceph public network. See https://github.com/ceph/ceph-docker/blob/c8c0bdf92a8bb866aef25ea691fd66502aead1a3/ceph-releases/jewel/ubuntu/14.04/daemon/entrypoint.sh#L221-L246.

File: examples/kubernetes/delete_ceph_cluster.sh (hunk 3 above)
Comment (font) [inline examples/kubernetes/delete_ceph_cluster.sh]: Haha yes it looks funny! This tells kubernetes to remove the `node-type` label from all nodes. The remove is indicated by the trailing `-`.

File: examples/kubernetes/ceph-mon-v1-ds.yaml (hunk 2 above)
Comment (leseb) [inline examples/kubernetes/ceph-mon-v1-ds.yaml]: Oh yes right, it makes sense now.

File: examples/kubernetes/delete_ceph_cluster.sh (hunk 3 above)
Comment (leseb) [inline examples/kubernetes/delete_ceph_cluster.sh]: Alright thanks for the explanation :)

**Pull request merged**